projects/dsp-ui/src/lib/viewer/operations/display-edit/display-edit.component.ts

```diff
@@ -254,7 +254,9 @@ export class DisplayEditComponent implements OnInit {
      * Show CRUD buttons and add 'highlighted' class to the element only if editModeActive is false
      */
     mouseEnter() {
-        this.showActionBubble = true;
+        if (this.canModify) {
```

Comment:
This is a general change, right?

Reply:
Can you clarify? :)

Reply:
oh, yeah, it's not really related to this PR but I figured it was a small enough change to sneak into this one. I can make a new PR if it's clearer.